README.md

```diff
@@ -24,7 +24,7 @@ migrate help # for more info
 
 Install golangci-lint with 
 
-	go get -u github.com/golangci/golangci-lint/cmd/golangci-lint
+    go get -u github.com/golangci/golangci-lint/cmd/golangci-lint github.com/securego/gosec
```

Comment:
Let's please fix the link

Reply:
> Let's please fix the link

Oh, I thought I fixed already here, too many repositories)